agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -317,19 +317,21 @@ main_thread::main_thread(config_file::sConfigSlave config_,
     auto now = std::chrono::steady_clock::now();
```

Comment:
Please remove `config_file::sConfigSlave config_` if its not used.

Reply:
but it is used